Using a backslash in the search causes a 500 error

```
play.api.UnexpectedException: Unexpected exception[IOException: not valid as a java.net.URI: https://preview.content.guardianapis.com/atoms?types=media&q=\&searchFields=data.title]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:276)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:206)
	at play.core.server.netty.PlayRequestHandler$$anonfun$2$$anonfun$apply$1.applyOrElse(PlayRequestHandler.scala:100)
	at play.core.server.netty.PlayRequestHandler$$anonfun$2$$anonfun$apply$1.applyOrElse(PlayRequestHandler.scala:99)
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:346)
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:345)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.api.libs.iteratee.Execution$trampoline$.execute(Execution.scala:70)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248)
	at scala.concurrent.Promise$class.complete(Promise.scala:55)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153)
	at scala.concurrent.Promise$class.failure(Promise.scala:104)
	at scala.concurrent.impl.Promise$DefaultPromise.failure(Promise.scala:153)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:257)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
	at akka.dispatch.BatchingExecutor$BlockableBatch$$anonfun$run$1.apply$mcV$sp(BatchingExecutor.scala:91)
	at akka.dispatch.BatchingExecutor$BlockableBatch$$anonfun$run$1.apply(BatchingExecutor.scala:91)
	at akka.dispatch.BatchingExecutor$BlockableBatch$$anonfun$run$1.apply(BatchingExecutor.scala:91)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:72)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:90)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:39)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:409)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: java.io.IOException: not valid as a java.net.URI: https://preview.content.guardianapis.com/atoms?types=media&q=\&searchFields=data.title
	at com.squareup.okhttp.Request.uri(Request.java:61)
	at com.squareup.okhttp.internal.http.RouteSelector.get(RouteSelector.java:78)
	at com.squareup.okhttp.internal.http.HttpEngine.connect(HttpEngine.java:324)
	at com.squareup.okhttp.internal.http.HttpEngine.sendRequest(HttpEngine.java:248)
	at com.squareup.okhttp.Call.getResponse(Call.java:273)
	at com.squareup.okhttp.Call$ApplicationInterceptorChain.proceed(Call.java:230)
	at com.squareup.okhttp.Call.getResponseWithInterceptorChain(Call.java:201)
	at com.squareup.okhttp.Call.execute(Call.java:81)
	at controllers.Support$$anonfun$previewCapiProxy$1$$anonfun$1$$anonfun$apply$1$$anonfun$apply$2.apply(Support.scala:33)
	at controllers.Support$$anonfun$previewCapiProxy$1$$anonfun$1$$anonfun$apply$1$$anonfun$apply$2.apply(Support.scala:24)
	at scala.Option.map(Option.scala:146)
	at controllers.Support$$anonfun$previewCapiProxy$1$$anonfun$1$$anonfun$apply$1.apply(Support.scala:24)
	at controllers.Support$$anonfun$previewCapiProxy$1$$anonfun$1$$anonfun$apply$1.apply(Support.scala:23)
	at scala.Option.flatMap(Option.scala:171)
	at controllers.Support$$anonfun$previewCapiProxy$1$$anonfun$1.apply(Support.scala:23)
	at controllers.Support$$anonfun$previewCapiProxy$1$$anonfun$1.apply(Support.scala:22)
	at scala.Option.flatMap(Option.scala:171)
	at controllers.Support$$anonfun$previewCapiProxy$1.apply(Support.scala:22)
	at controllers.Support$$anonfun$previewCapiProxy$1.apply(Support.scala:17)
	at play.api.mvc.ActionBuilder$$anonfun$apply$13.apply(Action.scala:371)
	at play.api.mvc.ActionBuilder$$anonfun$apply$13.apply(Action.scala:370)
	at com.gu.pandomainauth.action.AuthActions$AbstractApiAuthAction$class.invokeBlock(Actions.scala:323)
	at com.gu.pandomainauth.action.AuthActions$APIAuthAction$.invokeBlock(Actions.scala:278)
	at com.gu.pandomainauth.action.AuthActions$APIAuthAction$.invokeBlock(Actions.scala:278)
	at play.api.mvc.ActionBuilder$$anon$2.apply(Action.scala:458)
	at play.api.mvc.Action$$anonfun$apply$2$$anonfun$apply$5$$anonfun$apply$6.apply(Action.scala:112)
	at play.api.mvc.Action$$anonfun$apply$2$$anonfun$apply$5$$anonfun$apply$6.apply(Action.scala:112)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.mvc.Action$$anonfun$apply$2$$anonfun$apply$5.apply(Action.scala:111)
	at play.api.mvc.Action$$anonfun$apply$2$$anonfun$apply$5.apply(Action.scala:110)
	at scala.Option.map(Option.scala:146)
	at play.api.mvc.Action$$anonfun$apply$2.apply(Action.scala:110)
	at play.api.mvc.Action$$anonfun$apply$2.apply(Action.scala:103)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:253)
```

--- a/public/video-ui/src/actions/VideoActions/getVideos.js
+++ b/public/video-ui/src/actions/VideoActions/getVideos.js
@@ -66,10 +66,12 @@ function adaptCapiAtom(atom) {
 }
 
 export function searchVideosWithQuery(query) {
+  const encodedQuery = encodeURIComponent(query);
+
   return dispatch => {
     dispatch(requestSearchVideos());
 
-    return searchText(query)
+    return searchText(encodedQuery)
       .then(res => {
         const capiAtoms = res.response.results;
         const atoms = capiAtoms.map(adaptCapiAtom);

